packages/cozy-intent/package.json

```diff
@@ -15,9 +15,12 @@
   },
   "devDependencies": {
     "@babel/core": "7.12.3",
+    "@testing-library/react": "10.4.9",
     "babel-preset-cozy-app": "^2.0.1",
     "eslint-config-cozy-app": "^4.0.0",
-    "react": "17.0.2",
+    "jest": "26.2.2",
+    "mutationobserver-shim": "^0.3.7",
```

Comment:
i'm lost with all the different PR to review. I already add some reviews comments on another PRs. Could you close one of the two 🙏 ?

Reply:
That's my fault, I rebased my other PR on this branch and I forgot to move the Github pointer to it. So there were too many diffs. Your comments on https://github.com/cozy/cozy-libs/pull/1432 are related to this actual PR.

Reply:
![Capture d’écran 2022-01-28 à 11 23 11](https://user-images.githubusercontent.com/8363334/151530386-0c3acc00-5bf1-4ecb-804e-05d7c8f94c7e.png)